module/local/valheim.service

```diff
@@ -15,8 +15,5 @@ Group=${username}
 WorkingDirectory=/home/${username}/valheim
 ExecStartPre=/home/${username}/steam/steamcmd.sh +login anonymous +force_install_dir /home/${username}/valheim +app_update 896660 +quit
 ExecStart=/home/${username}/valheim/bootstrap_valheim.sh
-%{ if use_domain ~}
-ExecStopPost=/home/${username}/valheim/update_cname.sh
```

Comment:
I didn't need to add this as it's already included in the startup script [here](https://github.com/wahlfeld/valheim-on-aws/blob/main/module/local/start_valheim.sh#L16-L21). Also, you can't have more than one `ExecStopPost`, so I believe I broke the existing `ExecStopPost` from running, i.e., the one that's supposed to create a backup whenever the service is stopped.